screep

```diff
@@ -119,47 +120,42 @@ def sendData(channelName, data, params):
     chan.send(data, params)
 
 def encode(encoderNames, data, params):
```

Comment:
Holy out of scope variable Batman! encoderNames should be changed here to encoderName. Otherwise line 123 loads the variable defined whence it is called, which happens to exist. Shame on you Python! You call that encapsulation?!